samples/subsys/llext/shell_loader/README.rst

```diff
@@ -60,60 +60,89 @@ On a host machine with the zephyr sdk setup and the arm toolchain in PATH
 
-This extension can then be seen in the list of loaded extensions (`list`), its symbols printed (`list_symbols`), and the hello_world
-function which the extension exports can be called and run (`call_fn`).
+This extension can then be seen in the list of loaded extensions (`list`), its symbols printed
+(`list_symbols`), and the hello_world function which the extension exports can be called and
+run (`call_fn`).
 
 .. code-block:: console
 
   uart:~$ llext call_fn hello_world hello_world
   hello world
+  A number is 42
+
+In this sample there are 3 absolute (R_ARM_ABS32) relocations, 2 of which are mean to hold addresses into the .rodata sections where the strings are located. A third is an address of where the printk function (symbol) can be found. At load time llext replaces the values in the .text section with real memory addresses so that printk works as expected with the strings included in the hello world sample.
```

Comment:
```suggestion
In this sample there are 3 absolute (R_ARM_ABS32) relocations, 2 of which are meant to hold addresses into the .rodata sections where the strings are located. A third is an address of where the printk function (symbol) can be found. At load time llext replaces the values in the .text section with real memory addresses so that printk works as expected with the strings included in the hello world sample.
```